portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -58,21 +71,31 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        final String name = this.sanitizeHtml(request.getParameter("name"));
-        final String description = this.sanitizeHtml(request.getParameter("description"));
+        BlobRequestParser requestParser = new BlobRequestParser(request);
+
+        final String name = this.sanitizeHtml(requestParser.getTextParameter("name"));
+        final String description = this.sanitizeHtml(requestParser.getTextParameter("description"));
         final Long postDate = System.currentTimeMillis();
+        final List<BlobKey> imageFiles = requestParser.getFileListByParameter("images");
 
         try {
             if(!this.isAuthenticated()) {
                 throw new Exception();
             }
+            
+            List<String> imageUrls = new ArrayList<>();
+
+            for(BlobKey imageBlobKey : imageFiles) {
+                imageUrls.add(this.getUploadedUrl(imageBlobKey));
+            }
 
             CommentBuilder datastoreCommentBuilder = new CommentBuilder(CommentBuilderImplementationFactory.getDatastore());
 
             Entity newCommentEntity = (Entity)datastoreCommentBuilder
                 .setName(name)
                 .setDescription(description)
                 .setPostDate(postDate)
+                .setImages(imageUrls)
```

Comment:
Why is Images suddenly image Urls?

Rename Images in the Comment as Image URLs?